mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModelConfig.kt

```diff
@@ -0,0 +1,57 @@
+package com.airbnb.mvrx
+
+import androidx.annotation.RestrictTo
+import kotlinx.coroutines.CoroutineScope
+
+/**
+ * Provides configuration for a [BaseMavericksViewModel].
+ */
+abstract class MavericksViewModelConfig<S : Any>(
+    /**
+     * If true, extra validations will be applied to ensure the view model is used
+     * correctly.
+     */
+    val debugMode: Boolean,
+    /**
+     * The state store instance that will control the state of the ViewModel.
+     */
```

Comment:
This comment seems wrong

Reply:
@BenSchwab in what way? i can't find anything wrong